src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/state/attributesAccordionSelector.ts

```diff
@@ -1,47 +1,39 @@
 import { RootState } from 'src/store';
 
-export const getAttributes = (state: RootState): {} =>
-  state.customDownload.attributesAccordion.attributes;
+import Attributes, { AttributeWithOptions } from '../../../../types/Attributes';
+import JSONValue from 'src/shared/types/JSON';
+import { CheckboxGridOption } from 'src/content/app/custom-download/components/checkbox-grid/CheckboxGrid';
 
-export const getGeneAttributes = (state: RootState): {} =>
-  state.customDownload.attributesAccordion.attributes.gene;
+export const getAttributes = (state: RootState): Attributes =>
```

Comment:
Where is this selector used?

Reply:
Updated the following file to use it:
`src/content/app/custom-download/containers/content/attributes-accordion/sections/AttributesAccordionSection.tsx`